test/specs/modules/Popup/Popup-test.js

```diff
@@ -147,19 +147,28 @@ describe('Popup', () => {
 
         const rect = document.querySelector('.popup.ui').getBoundingClientRect()
         const { top, right, bottom, left } = rect
-
         expect(top).to.be.at.least(0)
         expect(left).to.be.at.least(0)
         expect(bottom).to.be.at.most(document.documentElement.clientHeight)
         expect(right).to.be.at.most(document.documentElement.clientWidth)
       })
-      it('is positioned properly when open property is set', () => {
+      it('is positioned properly when open property is set', (done) => {
         wrapperMount(<Popup content='_' position={position} open trigger={<button>foo</button>} />)
-        const element = document.querySelector('.popup.ui')
-        element.style.should.have.property('top', '')
-        element.style.should.have.property('left', '')
-        element.style.should.have.property('bottom', '')
-        element.style.should.have.property('right', '')
+        setTimeout(() => {
+          const element = document.querySelector('.popup.ui')
+          const { top, left, bottom, right } = element.style
+          expect(element.style.position).to.equal('absolute')
+          expect(top).to.not.equal('')
+          expect(left).to.not.equal('')
+          expect(bottom).to.not.equal('')
+          expect(right).to.not.equal('')
```

Comment:
Would be ideal if these made more specific assertions about where it should be positioned opposed to where it should not be positioned.

Reply:
Added new assertion `expect(element.parentNode).to.equal(document.body)`